src/main/Config.cpp

```diff
@@ -127,7 +127,7 @@ Config::Config() : NODE_SEED(SecretKey::random())
 
     MAXIMUM_LEDGER_CLOSETIME_DRIFT = 50;
 
-    OVERLAY_PROTOCOL_MIN_VERSION = 21;
+    OVERLAY_PROTOCOL_MIN_VERSION = 22;
```

Comment:
we can actually bump to 23 that corresponds to 19.3.0 released in July (this is in line with our release policy of 90 days)

Reply:
Updated!